src/stores/globals.ts

```diff
@@ -169,12 +172,33 @@ export const GlobalState = {
   state: () => ({
     loadingCounter: 0,
     objects: {},
+    parseObjects: {},
     feeds: {},
     defaultTeamId: (window as any) ? (window as any).AFFINITY_DEFAULT_TEAM : "",
     teamsBySlug: {},
     globalError: null,
   }),
   getters: {
+    // returns original parse object for id
+    parseObject<T extends TModel>(state: GlobalStateT) {
+      return (id: Parse.Object['id']) => state.parseObjects[id] as Parse.Object<T>;
+    },
+    // returns attributes of parse object for id
+    parseModel<T extends TModel>(_: GlobalStateT, getters: any) {
+      return (id: Parse.Object['id']) => {
+        const obj = { objectId: id, ...getters.parseObject(id)?.attributes } as T;
+        return Object.freeze(obj);
+      };
+    },
+    parseObjects<T extends TModel>(_: GlobalStateT, getters: any) {
+      return (ids: Parse.Object['id'][]) => ids?.map(id => getters.parseObject(id) as Parse.Object<T>);
+    },
+    parseModels<T extends TModel>(state: GlobalStateT, getters: any) {
+      return (ids: Parse.Object['id'][]) => (ids?.map(id => getters.parseModel(id) as T) || [] as T[]);
+    },
+    parseObjectsMap(state: GlobalStateT): Record<Parse.Object['id'], Parse.Object> {
+      return state.parseObjects;
+    },
```

Comment:
Why re-introduce the heavy `Parse.Object`s into the store again? The reason we convert them (right now) is that Parse.Objects.get isn't vuex-/reactify-observable (so UI doesn't update, when we update the objects) and that it the store (logs) transparent and introspectable...